layout/AppMenu.js

```diff
@@ -1,50 +1,90 @@
+  const onClose = () => showDialog("");
+  const { user } = useContext(UserContext);
+  const contextPath = getConfig().publicRuntimeConfig.contextPath;
+  const router = useRouter();
+  const model = [
+    {
+      items: [
+        { label: "Dashboard", icon: "pi pi-fw pi-chart-line", to: "/" },
+        { separator: true },
+        { label: "Clients", icon: "pi pi-fw pi-users", to: "/clients"},
+        { separator: true },
         {
-            label: 'Home',
-            items: [
-                { label: 'Dashboard', icon: 'pi pi-fw pi-home', to: '/' },
-                { label: 'Clients', icon: 'pi pi-fw pi-users', to: '/clients' },
-                { label: 'Animals', iconSrc: `${contextPath}/images/paw.svg`, to: '/animals' },
-                { label: 'Reports', icon: 'pi pi-fw pi-book', to: '/reports' },
-                { separator: true },
-                { 
-                  label: 'New Request', 
-                  iconSrc: `${contextPath}/images/pawClipboard.svg`, 
-                  key: 'newServiceRequest', 
-                  command: () => {if (!dialog) showDialog('newServiceRequest')} 
-                }
-            ]
-        }
-    ];
+          label: "Animals",
+          iconSrc:`${contextPath}/images/paw.svg`,
+          to: "/animals",
+          class: `animals-icon`,
+        },
+        { separator: true },
+        { label: "Reports", icon: "pi pi-fw pi-book", to: "/reports" },
+        { separator: true },
+        {
+          label: "New Request",
+          iconSrc:`${contextPath}/images/pawClipboard.svg`,
+          key: "newServiceRequest",
+          class:`new-request-icon`,
+          command: () => {
+            if (!dialog) showDialog("newServiceRequest");
+          },
+        },
+        { separator: true },
+        {
+          label: "Profile",
+          icon: "pi pi-user",
+          to: "/auth/login",
+          title: user?.email || "user",
+          command: () => {
+            signOut()
+          },
+        },
+      ],
+    },
+  ];
+
+     const signOut = (item) => {
+        authService.signOut().then(() => {
+            router.push("/auth/login")
+        })
+     };
+  
 
-    return (
-      <>
-        <MenuProvider>
-            <ul className="layout-menu">
-                {model.map((item, i) => {
-                    return !item.separator ? <AppMenuitem item={item} root={true} index={i} key={item.label} /> : <li className="menu-separator"></li>;
-                })}
-            </ul>
-        </MenuProvider>
-        <ServiceRequestDialog 
-          visible={dialog === 'newServiceRequest'}
-          onClose={onClose}
-        />
-      </>
-    );
+  return (
+    <>
+      <MenuProvider>
+        <ul className="layout-menu">
+          {model.map((item, i) => {
+            return !item.separator ? (
+              <AppMenuitem item={item} root={true} index={i} key={item.label} />
+            ) : (
+              <li className="menu-separator"></li>
```

Comment:
I made some changes due to the funky error we ran into during our session not sure if this is good enough if not be happen to change it. Since if we add a label on the separator it actually renders those labels instead of empty space.